precompiles/staking/StakingI.sol

```diff
@@ -238,6 +268,26 @@ interface StakingI is authorization.AuthorizationI {
             PageResponse calldata pageResponse
         );
 
+    /// @dev CreateValidator defines an Event emitted when a create a new validator.
+    /// @param delegatorAddress The address of the delegator
+    /// @param validatorAddress The address of the validator
+    /// @param commissionRate The commission rate charged to delegators, as a fraction
+    /// @param commissionMaxRate The commission max rate charged to delegators, as a fraction
+    /// @param commissionMaxChangeRate The commission max change rate charged to delegators, as a fraction
+    /// @param minSelfDelegation The validator's self declared minimum self delegation
+    /// @param pubkey The consensus public key of the validator
+    /// @param value The amount of coin being self delegated
+    event CreateValidator(
+        address indexed delegatorAddress,
+        string indexed validatorAddress,
+        uint256 commissionRate,
+        uint256 commissionMaxRate,
+        uint256 commissionMaxChangeRate,
+        uint256 minSelfDelegation,
+        string pubkey,
+        uint256 value
```

Comment:
This are not very relevant imo, we just need the validator address and the sender. @Vvaradinov wdyt?

Reply:
I think pubkey should be retained, because some people want to find relevant information through the public key?

Reply:
> I think pubkey should be retained, may be some people want to find relevant information through the public key?

I don't think anyone checks it from the Cosmos SDK event. So I'd skip it

Reply:
I'm in favor of keeping the events as leans as possible on the EVM side, considering also that the Cosmos Event for creating a validator only emits the validator address and the amount. So this event you defined should be the preferred one:

```solidity
event CreateValidator(
    address indexed delegatorAddress,
    address indexed validatorAddress,
    uint256 value
);
```

Reply:
If you had told me earlier, I would have changed it according to your instructions. But it’s okay, I’ll change it right away, hahaha.

Reply:
done !